pandas/io/sql.py

```diff
@@ -79,7 +79,9 @@ def _process_parse_dates_argument(parse_dates):
 
 def _handle_date_column(col, utc=None, format=None):
     if isinstance(format, dict):
-        return to_datetime(col, errors="ignore", **format)
+        return to_datetime(
+            col, **{"errors": "ignore", **format}  # type: ignore[call-overload]
```

Comment:
can you document what values format can actualy take on here? also don't type ignore.

Reply:
WDYM with "values format can take on here", should I write all the arguments that `to_datetime` accepts?

I added the type ignore since `to_datetime` is not overloaded for this argument combination and therefore results into a mypy error. I did not want to add an overload variation for this exception, so any tips on how to approach this?

Reply:
Updated the PR, is this what you meant with documenting values that format can take? Also changed the python code so that we dont have to type ignore anymore.